crates/teloxide-core/src/types/chat_permissions.rs

```diff
@@ -51,36 +51,67 @@ bitflags::bitflags! {
         /// locations and venues.
         const SEND_MESSAGES = 1;
 
-        /// Set if the user is allowed to send audios, documents,
-        /// photos, videos, video notes and voice notes, implies
+        /// Set if the user is allowed to send audios. implies
+        /// `SEND_MESSAGES`.
+        const SEND_AUDIOS = (1 << 1) | Self::SEND_MESSAGES.bits;
+
+        /// Set if the user is allowed to send documents. implies
+        /// `SEND_MESSAGES`.
+        const SEND_DOCUMENTS = (1 << 2) | Self::SEND_MESSAGES.bits;
+
+        /// Set if the user is allowed to send photos. implies
+        /// `SEND_MESSAGES`.
+        const SEND_PHOTOS = (1 << 3) | Self::SEND_MESSAGES.bits;
+
+        /// Set if the user is allowed to send videos. implies
+        /// `SEND_MESSAGES`.
+        const SEND_VIDEOS = (1 << 4) | Self::SEND_MESSAGES.bits;
+
+        /// Set if the user is allowed to send video notes. implies
+        /// `SEND_MESSAGES`.
+        const SEND_VIDEO_NOTES = (1 << 5) | Self::SEND_MESSAGES.bits;
+
+        /// Set if the user is allowed to send voice notes. implies
         /// `SEND_MESSAGES`.
-        const SEND_MEDIA_MESSAGES = (1 << 1) | Self::SEND_MESSAGES.bits;
+        const SEND_VOICE_NOTES = (1 << 6) | Self::SEND_MESSAGES.bits;
 
         /// Set if the user is allowed to send polls, implies
         /// `SEND_MESSAGES`.
-        const SEND_POLLS = (1 << 2) | Self::SEND_MESSAGES.bits;
+        const SEND_POLLS = (1 << 7) | Self::SEND_MESSAGES.bits;
+
+        /// Set if the user is allowed to send audios, documents,
+        /// photos, videos, video notes and voice notes, implies
+        /// `SEND_MESSAGES`, `SEND_AUDIOS`, `SEND_DOCUMENTS`,
+        /// `SEND_PHOTOS`, `SEND_VIDEOS`, `SEND_VIDEO_NOTES` and `SEND_VOICE_NOTES`.
+        const SEND_MEDIA_MESSAGES = (1 << 8) | Self::SEND_MESSAGES.bits
+                                            | Self::SEND_AUDIOS.bits
+                                            | Self::SEND_DOCUMENTS.bits
+                                            | Self::SEND_PHOTOS.bits
+                                            | Self::SEND_VIDEOS.bits
+                                            | Self::SEND_VIDEO_NOTES.bits
+                                            | Self::SEND_VOICE_NOTES.bits;
 
         /// Set if the user is allowed to send animations, games, stickers and
         /// use inline bots, implies `SEND_MEDIA_MESSAGES`.
-        const SEND_OTHER_MESSAGES = (1 << 3) | Self::SEND_MEDIA_MESSAGES.bits;
+        const SEND_OTHER_MESSAGES = (1 << 9) | Self::SEND_MEDIA_MESSAGES.bits;
 
         /// Set if the user is allowed to add web page previews to
         /// their messages, implies `SEND_MEDIA_MESSAGES`.
-        const ADD_WEB_PAGE_PREVIEWS = (1 << 4) | Self::SEND_MEDIA_MESSAGES.bits;
+        const ADD_WEB_PAGE_PREVIEWS = (1 << 10) | Self::SEND_MEDIA_MESSAGES.bits;
```

Comment:
```suggestion
        const ADD_WEB_PAGE_PREVIEWS = (1 << 10);
```

To account for `use_independent_chat_permissions`.